website/cue/reference/configuration.cue

```diff
@@ -252,6 +252,56 @@ configuration: {
 			}
 		}
 
+		telemetry: {
+			common: false
+			description: """
+				Configures options for how Vector emits telemetry.
+				"""
+			required: false
+			type: object: {
+				examples: []
+				options: {
+					tags: {
+						required: false
+						description: """
+							Controls which tags should be emitted with the `vector_component_sent_events_total` and
+							`vector_component_sent_event_bytes_total` metrics.
+						"""
+						type: object: {
+							examples: []
+							options: {
+								emit_source: {
+									common: true
+									description: """
+										Emit the name of the source component the event was received from. If there is
+										no source component, for example if the event was generated by the `lua` transform
+										a `-` is emitted for this tag.
+										"""
+									required: false
+									type: bool: {
+										default: false
+									}
+								}
+								emit_service: {
+									common: false
+									description: """
+										Emit the service the event was generated by. This is taken from a field of the event
```

Comment:
```suggestion
										Add a `service` tag with the service component the event was received from. If there is
```

For both service and source I think we should also call out how that data is found for logs, metrics, and traces.